.github/actions/find-all-charts-action/action.yml

```diff
@@ -0,0 +1,32 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2025 The Linux Foundation
+
+name: "Find all Helm charts"
+
+inputs:
+  COMPARISON_BRANCH:
+    description: "Branch to compare current code with"
+    required: false
+    default: "master"
+
+outputs:
+  CHARTS:
+    description: "List of charts that have changed from master"
+    value: ${{ steps.find_charts.outputs.charts }}
+  CHARTS_JSON:
+    description: "Json-formatted list of charts that have changed from master"
+    value: ${{ steps.find_charts.outputs.charts_json }}
+
+runs:
+  using: "composite"
```

Comment:
What is this about? Where does `composite` come from? I see it being called from all `action.yml` files

Reply:
https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action#composite-actions-and-reusable-workflows